Lib/test/test_concurrent_futures.py

```diff
@@ -759,6 +788,205 @@ def test_ressources_gced_in_workers(self):
                                        ProcessPoolForkserverMixin,
                                        ProcessPoolSpawnMixin))
 
+def hide_process_stderr():
+    import io
+    setattr(sys, "stderr", io.StringIO())
+
+
+def _crash():
+    """Induces a segfault"""
+    import faulthandler
+    faulthandler.disable()
+    faulthandler._sigsegv()
```

Comment:
I'm not sure whether this is reliable or not. Is `os.abort()` sufficient instead?
Perhaps @vstinner can help.

Reply:
`os.abort` dump traceback on `stderr` by default. This would make the test report unnecessarily noisy, so I would rather keep the current implementation.